# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/gfx-backend-dx11/shaders/clear.hlsl
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/gfx-backend-dx11/shaders/clear.hlsl b/third_party/rust/gfx-backend-dx11/shaders/clear.hlsl
deleted file mode 100644
--- a/third_party/rust/gfx-backend-dx11/shaders/clear.hlsl
+++ /dev/null
@@ -1,22 +0,0 @@
-cbuffer ClearColorF32   : register(b0) { float4 ClearF32; };
-cbuffer ClearColorU32   : register(b0) { uint4 ClearU32; };
-cbuffer ClearColorI32   : register(b0) { int4 ClearI32; };
-cbuffer ClearColorDepth : register(b0) { float ClearDepth; };
-
-// fullscreen triangle
-float4 vs_partial_clear(uint id : SV_VertexID) : SV_Position
-{
-    return float4(
-        float(id / 2) * 4.0 - 1.0,
-        float(id % 2) * 4.0 - 1.0,
-        0.0,
-        1.0
-    );
-}
-
-// TODO: send constants through VS as flat attributes
-float4 ps_partial_clear_float() : SV_Target0 { return ClearF32; }
-uint4  ps_partial_clear_uint() : SV_Target0 { return ClearU32; }
-int4   ps_partial_clear_int() : SV_Target0 { return ClearI32; }
-float  ps_partial_clear_depth() : SV_Depth { return ClearDepth; }
-void   ps_partial_clear_stencil() { }